src/modules.js

```diff
@@ -296,7 +296,8 @@ global.LibraryManager = {
           }
           const ret = sig == 'v' ? '' : 'return ';
           const args = genArgSequence(argCount).join(',');
-          lib[x] = new Function(args, `${ret}_${target}(${args});`);
+          const mangledName = mangleCSymbolName(target);
+          lib[x] = new Function(args, `${ret}${mangledName}(${args});`);
```

Comment:
Yes, without this you can't have the target of the alias be an internal JS function (i.e. one that starts with `$`).

I could rename the internal `$runtimeKeepalivePush` so that we don't need aliases at all but that would make the patch way bigger and should probable be a followup.